signer_with_wallet_example.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcd/txscript"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/btcsuite/btcwallet/frost"
+	"github.com/btcsuite/btcwallet/waddrmgr"
+	"github.com/btcsuite/btcwallet/wallet"
+	"os"
+	"runtime"
+	"time"
+)
+
+func main() {
```

Comment:
nit: by conventions it's better to locate examples in a separate dir, see https://github.com/golang-standards/project-layout/tree/master/examples

Reply:
the example uses the code from the main package, thus has to remain there (if we want to change the original code as little as possible)